suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -105,3 +105,10 @@ tests:
       module: rbd_trash.py
       name: Disable Trash and Delete images and check if they are not moved to trash automatically
       polarion-id: CEPH-83573296
+
+  - test:
+      desc: verify image doesnot exist after Moving images to trash and apply force purge to images
+      destroy-cluster: false
+      module: rbd_trash_purge_delete_trash_image.py
+      name: Move images to trash and apply force purge and check if they are not exist in trash
```

Comment:
Test name is too big it won't fit good in the mailing results,can you please make it short?
e.g `test Image trash force purge`

Reply:
done.